src/subdomains/core/payment-link/dto/create-payment-link-payment.dto.ts

```diff
@@ -29,4 +29,9 @@ export class CreatePaymentLinkPaymentDto {
   @IsDate()
   @Type(() => Date)
   expiryDate?: Date;
+
+  @ApiPropertyOptional()
+  @IsOptional()
+  @IsString()
+  webhookUrl?: string;
```

Comment:
- [ ] Add webhookUrl to payment entity
- [ ] Use URL from payment, if it exists

Reply:
My mistake, I understand the suggestion, but my intention was to use the webhook in the PaymentLink, not in the PaymentLinkPayment.

I updated the PR to reflect that so I think these suggestions no longer apply, thanks for the observation.

Reply:
Yes, good idea 👍